text/0008--individual-agency-company-pages.md

```diff
@@ -0,0 +1,131 @@
+
+#### Company Pages
+* Display list of companies using Gatsby to demonstrate it's "enterprise-ready" and fulfills whatever use-case they're looking for (e-commerce, authentication, etc). 
+* Help companies hire for Gatsby-related positions by surfacing their postings to the community; give the community a place to look for Gatsby-related jobs.
+
+# Detailed design
+
+## Pages
+
+#### All people / agencies / companies
+
+One template with multiple URLs, each of which apply different filters:
+  * /people/ (all people)
+  * /agencies/ (all agencies)
+  * /companies/ (all companies)
+
+Filter buttons should be at the top and should be easy to toggle between them (select-one). Filters should include:
+  * Individual 
+  * Agency 
+  * Company
+  * Open to new contracts (people & agencies with this field toggled)
+  * Now hiring (agencies & companies with this field toggled)
+
+#### Person / agency / company page
+
+One template with multiple URL patterns, depending on the Creator type:
+  * /companies/[company-name]
+  * /agencies/[agency-name]
+  * /people/[person-name]
+
+
+### Data Structures
+
+Showcase sites ([listed here](https://github.com/gatsbyjs/gatsby/blob/master/docs/sites.yml)) can have multiple Creators.
+
+**Creators** can either be an Individual, a Digital Agency, or a Company. . 
+
+Creator has the following fields:
+* an id, in slug format (eg mike-allanson)
+* type -- Individual, a Digital Agency, or a Company. We use the word Organization below to refer to either a Digital Agency and Company.
+* a description of them in the third person “Eg, Mike Allanson is a developer living in London"
+* Personal website
+* GH handle (?)
+* Open for Work [boolean]
+* A photo / logo (depending on Individual / Organization type)
+* Org they are affiliated with (if Individual)
+* People they are affiliated with (if Org)
+* A list of open jobs (if they are Organization type)
+* A list of all the sites they’ve built.
+* A location with:
+    * A description (“London, UK”)
+    * Lat / lng
+
+_Data Format_: all creators should live in a creators.yml file, similar to sites.yml.
+
+**Jobs** have the following fields:
+* Title
+* Location
+* Organization (FK)
+* (Optional) salary low end
+* (Optional) salary high end
+* (Optional) salary currency
+* Contact email or application link. 
+* Excerpt (short version)
+* Description
+
+_Data Format_: Jobs should be stored in markdown files in a /jobs folder. Jobs should be stored in markdown files. Frontmatter should have all fields except description. Body of the file should be the description. 
+
+### Notes on long-term data strategy
+* Right now, the data is in YAML in the GitHub repo. At some point, we may move all of our data to a format with a UI making it easier for people to submit & edit their own data, without imposing a review burden on Gatsby maintainers
```

Comment:
love this idea, was just thinking the same thing

Reply:
I talked about it quite a bit already, but I'll just second that I think we should talk about moving off YAML sooner rather than later.